examples/tina-cloud-starter/pages/api/test.ts

```diff
@@ -0,0 +1,17 @@
+import { NextApiHandler } from 'next'
+import { isAuthorized } from 'tina-cloud-next'
+const test: NextApiHandler = async (req, res) => {
+  const user = await isAuthorized(req, res)
```

Comment:
This example does not really make much sense (as it was for testing) so it could probably be removed before this is merged